R-package/README.md

```diff
@@ -116,12 +116,12 @@ You may also read [Microsoft/LightGBM#912](https://github.com/microsoft/LightGBM
 Examples
 --------
 
-Please visit [demo](demo):
-
-* [Basic walkthrough of wrappers](demo/basic_walkthrough.R)
-* [Boosting from existing prediction](demo/boost_from_prediction.R)
-* [Early Stopping](demo/early_stopping.R)
-* [Cross Validation](demo/cross_validation.R)
-* [Multiclass Training/Prediction](demo/multiclass.R)
-* [Leaf (in)Stability](demo/leaf_stability.R)
-* [Weight-Parameter Adjustment Relationship](demo/weight_param.R)
+Please visit [demo](https://github.com/microsoft/LightGBM/tree/master/R-package/demo):
+
+* [Basic walkthrough of wrappers](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/basic_walkthrough.R)
+* [Boosting from existing prediction](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/boost_from_prediction.R)
+* [Early Stopping](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/early_stopping.R)
+* [Cross Validation](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/cross_validation.R)
+* [Multiclass Training/Prediction](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/multiclass.R)
+* [Leaf (in)Stability](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/leaf_stability.R)
+* [Weight-Parameter Adjustment Relationship](https://github.com/microsoft/LightGBM/blob/master/R-package/demo/weight_param.R)
```

Comment:
@jameslamb Thanks for catching this. Indeed, this should be reverted, but **after** we start to  generate demos on RTD site. At present, due to that the demo part is not built by `pkgdown` and it converts `README` to `index.html` (it can be modified by creating `INDEX.md` file in the repo, BTW), these links lead to non-existent pages on RTD. So, we should choose now: relative links in the repo and broken links on RTD, or absolute working links to GitHub.

```
# # to-do 
# build_articles(preview = FALSE) 
# build_tutorials(preview = FALSE)
```

![image](https://user-images.githubusercontent.com/25141164/57981606-26450b00-7a42-11e9-8ae5-c03cca0236a3.png)